Sources/Widgets/Widgets3D/SplineWidget/behavior.js

```diff
@@ -23,6 +23,30 @@ export default function widgetBehavior(publicAPI, model) {
 
   // --------------------------------------------------------------------------
 
+  publicAPI.setPixelScale = (pixelScale) => {
+    model.pixelScale = pixelScale;
+  };
+
+  // --------------------------------------------------------------------------
+
+  publicAPI.updateHandlesSize = () => {
+    if (model.scaleHandlesInPixelSpace) {
```

Comment:
just checking the value of pixelScale should be enough (e.g. `if model.pixelScale !== null`)